File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.go
func getRow(rowIndex int) []int {
    ans := make([][]int, rowIndex + 1)
    
    for row := 0; row <= rowIndex; row++ {
        ans[row] = make([]int, row + 1)
        ans[row][0] = 1
        ans[row][row] = 1
        for col := 1; col < row; col++ {
            ans[row][col] = ans[row - 1][col - 1] + ans[row - 1][col]
        }
    }
    
    return ans[rowIndex]
}

File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.php
class Solution {

    /**
     * @param Integer $rowIndex
     * @return Integer[]
     */
    function getRow($rowIndex) {
        $ans = [1];
        
        if($rowIndex === 0){
            return $ans;
        }
        
        for ($row = 1; $row <= $rowIndex; $row++){
            $ans[$row][0] = 1;
            for($col = 1; $col < $row; $col++) {
                $ans[$row][$col] = $ans[$row - 1][$col - 1] + $ans[$row - 1][$col];
            }
            
            $ans[$row][] = 1;
        }
        
        return $ans[$rowIndex];
    }
}